sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2835,6 +2835,82 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
     checkAnswer(df, Row(1, 3, 4) :: Row(2, 3, 4) :: Row(3, 3, 4) :: Nil)
   }
 
```

Comment:
I think we need more exhaustive tests for supporting group filter. Could you add tests in `SQLQueryTestSuite`, e.g., input/group-by-filter.sql`?

Reply:
OK. I will added this in new PR.